src/test/java/core/basesyntax/strategy/PurchaseHandlerTest.java

```diff
@@ -0,0 +1,54 @@
+package core.basesyntax.strategy;
+
+import static org.junit.Assert.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import core.basesyntax.db.Storage;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class PurchaseHandlerTest {
+
+    private OperationStrategy purchaseHandler = new PurchaseHandler();
+
+    @BeforeEach
+    void setUp() {
+        Storage.inventory.clear();
+        purchaseHandler = new PurchaseHandler();
+        Storage.inventory.put("apple", 10);
```

Comment:
do you really need it before each? Better write it in test case that require data preparing